### ISSUE TITLE
deprecated "rnpm"

### ISSUE THREAD

**Issue by rochapablo** [rhdeck/react-native-swift#3]
After update to "react-native": "0.60.0", I'm getting

```
warn The following packages use deprecated "rnpm" config that will stop working from next release:
  - react-native-swift: https://github.com/rhdeck/react-native-swift#readme
Please notify their maintainers about it. You can find more details at https://github.com/react-native-community/cli/blob/master/docs/configuration.md#migration-guide.
```

**Comment by rochapablo** [comment]
https://github.com/rhdeck/react-native-swift/pull/2

**Comment by AsheeshPandey1995** [comment]
same here>> The following packages use deprecated "rnpm" config that will stop working from next release
react-native-swift: https://github.com/rhdeck/react-native-swift#readme
Please notify their maintainers about it. You can find more details at https://github.com/react-native-community/cli/blob/master/docs/configuration.md#migration-guide.

**Comment by pavm035** [comment]
Same here any updates?

**Comment by mmrk09** [comment]
I'm still getting this error, which version do I need to upgrade to in order to resolve?

**Comment by rhdeck** [comment]
Fixed in 2.0
